apps/studio/src/stories/Page/SignInPage/EmailWithSingpassSignInPage.stories.tsx

```diff
@@ -0,0 +1,75 @@
+import type { Meta, StoryObj } from "@storybook/react"
+import { expect, userEvent, within } from "@storybook/test"
+import { authEmailHandlers } from "tests/msw/handlers/auth/email"
+import { meHandlers } from "tests/msw/handlers/me"
+
+import { withChromaticModes } from "@isomer/storybook-config"
+
+import SignInPage from "~/pages/sign-in"
+import { createSingpassEnabledGbParameters } from "~/stories/utils/growthbook"
+
+const VALID_AUTH_EMAIL = "test@example.gov.sg"
+
+const meta: Meta<typeof SignInPage> = {
+  title: "Pages/Sign In Page/Email with Singpass Sign In Page",
+  component: SignInPage,
+  parameters: {
+    loginState: false,
+    growthbook: [createSingpassEnabledGbParameters(true)],
+    msw: {
+      handlers: [
+        meHandlers.unauthorized(),
+        authEmailHandlers.login({
+          email: VALID_AUTH_EMAIL,
+          otpPrefix: "TST",
+        }),
+        authEmailHandlers.verifyOtp.default(),
+      ],
+    },
+  },
+}
+
+export default meta
+type Story = StoryObj<typeof SignInPage>
+
+export const Default: Story = {
+  parameters: {
+    chromatic: withChromaticModes(["gsib", "mobile"]),
+  },
```

Comment:
maybe move it into `const meta: Meta<typeof SignInPage>`

Reply:
d31f2e21cec1b59475be7c2903ead8357b041322